catalyst-gateway/tests/Earthfile

```diff
@@ -41,7 +41,7 @@ fuzzer-api:
         --compose schemathesis-docker-compose.yml \
         --load schemathesis:latest=(+package-schemathesis --openapi_spec=$OPENAPI_SPEC) \
         --load event-db:latest=(../event-db+build) \
-        --load cat-gateway:latest=(../+package-cat-gateway --address=$CAT_ADDRESS --db_url=$DB_URL) \
```

Comment:
If you are passing the env variables via docker-compose remove also this two lines
```
ARG DB_URL="postgres://catalyst-event-dev:CHANGE_ME@localhost/CatalystEventDev"
ARG CAT_ADDRESS="127.0.0.1:3030"
```